### PR TITLE
related people to groups

### PR DESCRIPTION
**Pull request by dovebair**
created membership model and related people and groups to it and eachother. Not sure if this even makes sense schema wise

### CODE REVIEW

File: app/models/group.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Group < ApplicationRecord
```
Comment (jaggederest) [inline app/models/group.rb]: `has_many :memberships`
`has_many :users, through: :memberships`

File: app/models/person.rb [hunk 2]
```diff
@@ -1,4 +1,3 @@
 class Person < ApplicationRecord
-  has_many :consumptions
-  
+  has_many :groups
```
Comment (jaggederest) [inline app/models/person.rb]: No longer has groups

File: app/models/role.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class Role < ApplicationRecord
```
Comment (jaggederest) [inline app/models/role.rb]: `has_many :memberships`
`has_many :users, through: :memberships`

**Comment by dovebair** [comment]
@jaggederest this is ready for re-review

**Pull request merged**